File: ios/WaveBridge.m
#import <React/RCTBridgeModule.h>
#import <Foundation/Foundation.h>
#import <UIKit/UIKit.h>
#import <React/RCTLog.h>

@interface RCT_EXTERN_MODULE(WaveBridge, NSObject)

RCT_EXTERN_METHOD(addEvent:(NSString *)name location:(NSString *)location date:(nonnull NSNumber *)date)

// RCT_EXTERN_METHOD(getBack:(NSString *)name callback:(RCTResponseSenderBlock)callback)

@end

File: ios/WaveBridge.swift
//
//  WaveBridge.swift
//  RNMokam
//
//  Created by Jongwoo Moon on 2017. 12. 17..
//  Copyright © 2017년 Facebook. All rights reserved.
//

import Foundation
import UIKit



@objc(WaveBridge)
class WaveBridge: NSObject {
    var rctCB: RCTResponseSenderBlock? = nil
    var connCB: (() -> ())? = nil
    @objc(addEvent:location:date:)
    func addEvent(name: String, location: String, date: NSNumber) -> Void {
        print("  add event in swift ::  \(name),  \(location),  \(date) ")
    }
    
}
